src/plugins/github/declaration.js

```diff
@@ -59,7 +59,7 @@ export const userNodeType = deepFreeze({
   name: "User",
   pluralName: "Users",
   prefix: N.Prefix.user,
-  defaultWeight: 1,
+  defaultWeight: 0,
   description: "NodeType for a GitHub user",
```

Comment:
Seems like we should apply this for GitHub bots as well.

Reply:
Done, thanks.